src/Location/Location.tsx

```diff
@@ -1,10 +1,92 @@
 import React from 'react'
 import './Location.scss'
+import cloudyNight from '../assets/02n.png'
+import waterdrop from '../assets/water-drop.png'
+import eye from '../assets/eye.png'
+import sun from '../assets/sun.png'
+import bee from "../assets/bee.png";
+import lungs from '../assets/lungs.png'
+import wind from '../assets/wind.png'
+// import raindrop from '../assets/liquid.png'
+// should we break this out into different components?
+// feel free to break up lines of text for readability, i'm getting tired
 
 const Location: React.FC = () => {
 	return (
-		<section className="location-data">
-			<p></p>
+		<section className="location-section">
```

Comment:
THIS IS AWESOME! I got fetch working last night with the new API and passing props from the searched result from App into the various components, including this one! I will keep these details in mind when I am pulling data and see if there's anything else we could add as well!